locales/English/gamecreate.php

```diff
@@ -347,4 +384,45 @@ function setBotFill(){
 		document.getElementById("botBox").checked = false;
 	}
 }
+
+// Display nextPhaseMinutes paragraph only if phaseSwitchPeriod has selected a period.
+nextPhaseMinutesPara = document.getElementById("nextPhaseMinutesPara");
+
+selectPhaseSwitchPeriod = document.getElementById("selectPhaseSwitchPeriod");
+phaseSwitchPeriodPara = document.getElementById("phaseSwitchPeriodPara");
+
+selectPhaseMinutes = document.getElementById("selectPhaseMinutes");
+
+nextPhaseMinutesPara.style.display = "none";
+phaseSwitchPeriodPara.style.display = "none";
+
+
+const updatePhasePeriod = () => {

```

Comment:
Unfortunately some of the arrow function logic here isn't going to work correctly on older phones and browsers, we don't have a back compiler to downgrade from es6 to lower versions so things like arrow functions and const/let don't function. Would you mind rewriting these with base js functionality?

Reply:
Done